mkdocs.yml

```diff
@@ -174,6 +174,8 @@ plugins:
           merge_init_into_class: true
```

Comment:
Could be nice to add these too:

```
parameter_headings: true
show_signature_annotations: true
signature_crossrefs: true
```

Reply:
For parameter headings, you'll have to install the Insiders version of mkdocstrings-python, similar to how you install mkdocs-material:

```
pdm run python -m pip install https://files.scolvin.com/${MKDOCS_TOKEN}/mkdocs_material-9.4.2+insiders.4.42.0-py3-none-any.whl
```

https://mkdocstrings.github.io/python/insiders/installation/

Let me know if any other user needs to be added to the @pawamoy-insiders organization :slightly_smiling_face:

Reply:
Added:

```yml
show_signature_annotations: true
signature_crossrefs: true

```